README.md

```diff
@@ -59,6 +59,15 @@ $ git checkout {target branch}
 $ mvn -U install -am -pl terasoluna-gfw-functionaltest-web
 ```
 
+> **Note:**
+> 
+> When using JDK 7, build as follows.
+> you must compile with the target version 1.7 in accordance with the runtime environment JVM.
+> You can set the target version using `java-version` property.
+> ```console
+> $ mvn -U install -am -pl terasoluna-gfw-functionaltest-web -D java-version=1.7
```

Comment:
- [ ] Wrong Specifying -D option.

not `-D java-version=1.7`
but `-Djava-version=1.7`